docs/specs/openapi.yaml

```
@@ -0,0 +1,335 @@
+              value: svc1
+              summary: one service name
+            two:
+              value: svc1,svc2
+              summary: multiple services names, comma separated
+      responses:
+        '200':
+          description: List of services
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/GetServices"
+              examples:
+                list all services:
+                  value:
+                    - type: sync
+                      status-code: 200
+                      status: OK
+                      result:
+                        - name: "svc1"
+                          startup: "enabled"
+                          current: "active"
+                          current-since: "2024-12-11T20:10:33.381993977+08:00"
+                        - name: "svc2"
+                          startup: "disabled"
+                          current: "stopped"
+    post:
+      summary: Manage services
+      description: Services operations, like start/stop/restart a service/services, auto-start default services, and replan.
+      tags:
+        - services
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                action:
+                  type: string
+                  description: The action to perform. One of "start", "stop", "restart", "replan", "autostart".
+                  enum: ["start", "stop", "restart", "replan", "autostart"]
+                services:
+                  type: array
+                  description: |
+                    A list of service names.  Required for "start", "stop", and "restart".
+                    Ignored for "replan" and "autostart" (resolved automatically for "autostart" to default services).
+                  items:
+                    type: string
+            examples:
+              start one:
+                value:
+                  action: start
+                  services: ["svc1"]
+              stop multiple:
+                value:
+                  action: stop
+                  services: ["svc1", "svc2"]
+              restart multiple:
+                value:
+                  action: restart
+                  services: ["svc1", "svc2"]
+              replan:
+                value:
+                  action: replan
+              autostart:
+                value:
+                  action: autostart
+      responses:
+        '202':
+          description: Accepted - asynchronous operation started. Returns a change ID.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/PostServices"
+              examples:
+                accepted:
+                  value:
+                    type: async
+                    status-code: 202
+                    status: Accepted
+                    change: 1
```

This is always a string, so:
```suggestion
                    change: "1"
```